### PR TITLE
chore: restore README.md

### PR DESCRIPTION
**Pull request by livefast-eattrash**
This PR restores the assets which were included in the old `README.md`. Moreover, it updates some of the textual information contained in this file too, e.g. the references to Moko resources which is not used any more.

The current `README.md` was taken from a snapshot of an older version, so the features section was not entirely up-to-date: I've tried to keep the same synthetic style but mention all the features that have been added since the time the snapshot was taken.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,86 +61,86 @@ Please be willing to contribute if you can, instead of being ready to demand. Th
...
-Moreover, developing a new client was an opportunity to add all the features that I needed and that
-were "scattered" across different apps, e.g. I liked the feature richness
-of [Liftoff](https://github.com/liftoff-app/liftoff) (e.g. the possibility to explore all the
-communities of an external instance in guest mode), the multi-community feature of
-[Summit](https://github.com/idunnololz/summit-for-lemmy) and the polished UI of the really great
-[Thunder](https://github.com/thunder-app/thunder) and I wished I could have them all in the same
-app.
+In the third place, this app has been a means to dig deeper inside Lemmy's internals and become more
+humble and patient towards other apps because there are technical difficulties in having to deal
+with a platform likeLemmy.
```
Comment (N7-X) [inline README.md:127]: One missing space, but otherwise it all looks beautiful! 
I like the other changes.

**Review by N7-X — CHANGES_REQUESTED** [review]
Aside from the one dropped space, all looks flawless to my eyes!

File: README.md (hunk 1 above)
Comment (AkesiSeli) [reply README.md]: Ok, done!

**Comment by N7-X** [comment]
Using squash and merge, as discussed.  Thank you!

**Pull request merged**